templates/experiments/experiment_form.html

```diff
@@ -122,6 +122,17 @@ <h3 class="text-md font-bold">{% translate "Please note" %}</h3>
       </div>
     </div>
 {% endblock form %}
+{% block form_actions %}
+  <input type="submit" class="pg-button-primary mt-2" value="{{ button_text }}">
+  {% flag "experiment_versions" %}
+    {% if show_version_action_options %}
```

Comment:
I think this if statement isn't necessary now that this is in the experiment form template